core/state/statedb.go

```diff
@@ -362,6 +362,26 @@ func (s *StateDB) GetCodeHash(addr common.Address) common.Hash {
 	return common.Hash{}
 }
 
+// ResolveCode retrieves the code at addr, resolving any delegation designations
+// that may exist.
+func (s *StateDB) ResolveCode(addr common.Address) []byte {
+	stateObject := s.resolveStateObject(addr)
+	if stateObject != nil {
+		return stateObject.Code()
+	}
+	return nil
+}
+
+// ResolveCodeHash retrieves the code at addr, resolving any delegation
+// designations that may exist.
+func (s *StateDB) ResolveCodeHash(addr common.Address) common.Hash {
```

Comment:
wdyt about making GetCodeHash an alias of ResolveCodeHash, seems like we almost never use GetCodeHash anymore, maybe we don't need the two behaviors. (we do need them for getCode/resolveCode, but maybe not for the codehash)

Reply:
Since we have separate `GetCode / ResolveCode`, I think it makes sense to also have separate `GetCodeHash / ResolveCodeHash`... (?)